Engine/Engine/Math/Point/math_point.h

```diff
@@ -0,0 +1,82 @@
+/*
+** EPITECH PROJECT, 2023
+** Title: Engine-3D
+** Author: MasterLaplace
+** Created: 2023-10-12
+** File description:
+** math_point
+*/
+
+#ifndef MATH_POINT_H_
+    #define MATH_POINT_H_
+
+////////////////////////////////////////////////////////////
+// Include the other headers of the math module
+////////////////////////////////////////////////////////////
+#include "config.h"
+
+////////////////////////////////////////////////////////////
+// Include necessary headers for the point module
+////////////////////////////////////////////////////////////
+#include <math.h>
+
+////////////////////////////////////////////////////////////
+// Define the default point of floats
+////////////////////////////////////////////////////////////
+#define POINT_F_DEFAULT (point_f){0, 0}
```

Comment:
Not sure about all the "default" state macros, since you're editing all fields anyway, sounds like it adds a bit of mental gymnastic to the reader, as `{ 0 }` is already quite clear.